design-toolkit/components/src/components/classification-banner/styles.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from 'tailwind-variants';
+
+export const ClassificationBannerStylesDefaults = {
+  variant: 'missing',
+} as const;
+
+// Classification banner default empty content per variant
+export const CLASSIFICATION_BANNER_EMPTY_LABELS = {
```

Comment:
These are nearly identical to the badge ones. Can they be the same? Question for design(?) or POs(?) @brandonjpierce may have an opinion

Reply:
yup, was thinking about using the same ones, but figured there may be a case where the chips say something and the banner say something else so kept two diff dictionaries for that reason